input/templates/partials/catalog/quickview.hbs

```diff
@@ -1,10 +1,11 @@
+<!-- start catalog/quickview.hbs -->
 <div class="modal fade" id="quickview-modal{{index}}" tabindex="-1" role="dialog" aria-labelledby="quickviewModal{{index}}">
   <div class="modal-dialog modal-dialog-quickview" role="document">
     <div class="modal-content quickview-modal">
       <div class="modal-header">
         <button type="button" class="close" data-dismiss="modal" aria-label="{{i18n "main:close"}}">
           <span aria-hidden="true">
-            <img src="{{@root.meta.assetsPath}}img/close.png" alt="{{i18n "main:close"}}" />
+            <img src="{{@root.meta.assetsPath}}img/close-4fd7b89129be75523cd0ca2b8e3e5368.png" alt="{{i18n "main:close"}}" />
```

Comment:
You should remove the fingerprint from everywhere, Grunt should fill the correct fingerprint for you during the release process :)